docs/rss-tutorials/state_estimation_tutorial_preparations.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: State Estimation Tutorial
+parent: Summer School Tutorials
+---
+
+# Preparations for State Estimation Tutorial
+For the tutorial you need the following packages:
+ - [ethzasl_msf](https://github.com/ethz-asl/ethzasl_msf)
+ - [compslam](https://bitbucket.org/leggedrobotics/compslam_rss/src/master/)
+
+You can test it on your own computer by running it with a rosbag provided [bellow](#Download files).
+
+## Installation
+Download the repositories and build them:
```

Comment:
Please extend these to account for the fact that the repositories needs to be added to the catkin workspace containing the smb_core environment .

Reply:
Thanks, this should be specified.